utils/travis/common.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+
+export CPUS_USED=2
+if [[ $(command -v nproc) != "" ]]; then
```

Comment:
What about `if command -v nproc`; then?

Reply:
I think this outputted me the command path then, when I executed the script.

Reply:
append `>/dev/null 2>&1`, so it's `if command -v nproc >/dev/null 2>&1; then`

Reply:
But that makes the code more complicated than necessary.